### PR TITLE
Downloads & Selections

### PR DESCRIPTION
**Pull request by stdavis**
Implement selections and downloads. The performance really suffers when there are hundreds of results. It appears to be in the Table. I'm going to wait until we switch to the UDS table before I spend too much time on it.

### CODE REVIEW

File: src/components/ResultsGrid.tsx [hunk 1]
```diff
@@ -125,67 +137,110 @@ export default function ResultsGrid() {
...
+          </span>
+        )}
+      </span>
+      <Tabs aria-label="results panel">
+        <TabList>
+          <Tab id="grid">Results</Tab>
+          <Tab id="download">Download</Tab>
+        </TabList>
+        <TabPanel id="grid">
+          <Table
```
Comment (steveoh) [inline src/components/ResultsGrid.tsx:175]: **thought**: I believe every time the tabs are switched the content is reloaded and any state is lost. 

Can you explain the steps to identify the performance issue you talk about?

Would the tab state be related?

**Review by steveoh — APPROVED** [review]
looks good. It's probably not the best idea to have a table in a tab the way it's implemented now since it detaches on change and all state etc is lost. 

**edit** it appears the table state is preserved so nevermind.

File: src/components/ResultsGrid.tsx (hunk 1 above)
Comment (stdavis) [reply src/components/ResultsGrid.tsx:175]: I'm not sure why there is a performance issue. It's much worse in development than in the production build. I just added a story to demonstrate the issue and show that it's specific to the table component.

Comment (stdavis) [reply src/components/ResultsGrid.tsx:175]: I've tried profiling in dev tools and ran into this bug: https://github.com/facebook/react/issues/25415

Comment (steveoh) [reply src/components/ResultsGrid.tsx:175]: I'm not fixed to this table implementation. With the amount of experience we have with tanstack table, I'd be happy to use that if you think it makes more sense.

Comment (stdavis) [reply src/components/ResultsGrid.tsx:175]: https://github.com/adobe/react-spectrum/issues/5707

**Pull request merged**